docs/.vuepress/public/arrow-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="9" viewBox="0 0 16 9">
```

Comment:
Why is it added into the project, this is not used directly

Reply:
These are icons I used here https://monosnap.com/file/Ktp9XxizkkZEeYweNQM6KGGSjlQyNe. I put them there to have all assets in the same place. I'll move to the theme repo if it's not acceptable.